tests/visual/dateinputs.html

```diff
@@ -35,6 +35,12 @@
         <span class="k-widget k-datetimepicker"><span class="k-picker-wrap k-state-hover"><input type="text" class="k-input" value="Hovered" /><span class="k-select"><span class="k-link k-link-date"><span class="k-icon k-i-calendar"></span></span><span class="k-link k-link-time"><span class="k-icon k-i-clock"></span></span></span></span></span>
         <span class="k-widget k-dateinput"><span class="k-dateinput-wrap k-state-hover"><input type="text" class="k-input" value="Hovered" /><span class="k-select"><span class="k-link k-link-increase"><span class="k-icon k-i-arrow-n"></span></span><span class="k-link k-link-decrease"><span class="k-icon k-i-arrow-s"></span></span></span></span></span>
 
+        <!-- jQuery - hovered button  -->
+        <span class="k-widget k-datepicker"><span class="k-picker-wrap k-state-hover"><input type="text" class="k-input" value="Hovered button" /><span class="k-select"><span class="k-icon k-i-calendar"></span></span></span></span>
+        <span class="k-widget k-timepicker"><span class="k-picker-wrap k-state-hover"><input type="text" class="k-input" value="Hovered button" /><span class="k-select"><span class="k-icon k-i-clock"></span></span></span></span>
```

Comment:
The buttons of these pickers are not "hovered".

Reply:
Because there is no k-link there, k-select already has hovered effect